pizzeria-veritas.json

```diff
@@ -0,0 +1,24 @@
+# Pizzeria Veritas
```

Comment:
Please delete all these non-JSON lines... even the comments. JSON is very picky and doesn't understand comments.